ghost/signup-form/vite.config.ts

```diff
@@ -42,7 +49,9 @@ export default (function viteConfig() {
                 output: {}
             },
             commonjsOptions: {
-                include: [/packages/, /node_modules/]
+                include: [/ghost/, /packages/, /node_modules/],
+                dynamicRequireRoot: '../',
+                dynamicRequireTargets: SUPPORTED_LOCALES.map(locale => `../i18n/locales/${locale}/portal.json`)
```

Comment:
```suggestion
                dynamicRequireTargets: SUPPORTED_LOCALES.map(locale => `../i18n/locales/${locale}/signup-form.json`)
```

Reply:
Just for your knowledge, this bit of config means we only bundle the `signup-form.json` files with signup-form, and not the portal.json/ghost.json ones 😀